### ISSUE TITLE
Can't override title in base template

### ISSUE THREAD

**Issue by awgreenblatt** [marcgibbons/django-rest-swagger#668]
Hi there -

I created a simple pull request to solve this.  I want to override the <title> tag in base.html, but the only way currently to do so is to completely replace base.html, which is a bit overkill. I added a block around the title.

Thanks very much,

Alan

**Comment by lwm** [comment]
This should probably be set to the same value as `title + 'Swagger UI'` in `get_swagger_view(title='my title')` by default **and** overridable, no?

**Comment by awgreenblatt** [comment]
Agreed, the title in base.html should default to whatever was set in get_swagger_view(), but to me that's a separate issue. I'm not personally sure how to get the title from the schema into the renderer context so it could be used in the template. I'd personally rather if we kept the two separate so we didn't hold up being able to override the template. Unless of course, someone else wants to provide the code to initialize the title in base.html to that set in get_swagger_view().

**Comment by lwm** [comment]
>  I'd personally rather if we kept the two separate so we didn't hold up being able to override the template.

:+1:, hope this gets in soon!

**Comment by neilhut42** [comment]
A template override block on base.html would be a good start :-)
